lib/ecto/schema.ex

```diff
@@ -1369,11 +1369,11 @@ defmodule Ecto.Schema do
     * `:join_where` - A filter for the join table. See "Filtering associations"
       in `has_many/3`
 
-    * `:preload_order` - Sets the default `order_by` of the association.
-      It is used when the association is preloaded.
-      For example, if you set `Post.many_to_many :tags, Tag, join_through: "posts_tags", preload_order: [asc: :foo]`,
-      whenever the `:tags` associations is preloaded, the tags will be order by the `:foo` field.
-      See `Ecto.Query.order_by/3` for more examples.
+    * `:preload_order` - Sets the default `order_by` when preloading the association.
+      It may be a keyword list/list of fields or an MFA tuple, such as `{Mod, fun, []}`.
+      Both cases must resolve to a valid `order_by` expression. See `Ecto.Query.order_by/4`
+      to learn more about about ordering expressions. See the preload order section below to
```

Comment:
@josevalim Is there a way with ex doc to link to sections within the same function doc? Or if not is this something that is theoretically possible and I could investigate adding?

Reply:
It is possible to link to h2 (maybe h3). Run `MIX_ENV=docs mix docs` and see the link. :)

Reply:
ah beautiful, thank you!